frontend/javascripts/components/fixed_expandable_table.tsx

```diff
@@ -12,7 +12,10 @@ type State = {
  *  and the scroll prop as this is already done by the wrapper.
  */
 
-export default class FixedExpandableTable extends React.PureComponent<TableProps, State> {
+type OwnTableProps<RecordType = any> = TableProps<RecordType> & {
```

Comment:
```suggestion
// Enforce the "columns" prop which is optional by default otherwise 
type OwnTableProps<RecordType = any> = TableProps<RecordType> & {
```